### PR TITLE
Next compat

### PR DESCRIPTION
**Pull request by shentao**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [x] Refactor
- [x] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `dev` branch for v1.x (or to a previous version branch), _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] All tests are passing: https://github.com/vuelidate/vuelidate/blob/master/.github/CONTRIBUTING.md#development-setup
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: packages/vuelidate-compat/jest.config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const base = require('../../jest.base')
+
+module.exports = {
+  ...base,
+  name: 'Vuelidate',
```
Comment (dobromir-hristov) [inline packages/vuelidate-compat/jest.config.js:5]:
```suggestion
  name: 'Vuelidate Compat',
```

Maybe like this?

File: packages/vuelidate-compat/jest.config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  collectCoverage: true,
+  collectCoverageFrom: ['<rootDir>/src/**/*.js'],
+  displayName: 'Vuelidate'
```
Comment (dobromir-hristov) [inline packages/vuelidate-compat/jest.config.js:8]: here as well

**Review by dobromir-hristov — COMMENTED** [review]
This is not OK imho, we cant maintain this at all, if we copy the entire library :/ One will play catch with the other all the time.

 I dont know how we can re-architect this to work nicely for both Vue versions, without conflicts in node_modules (as I am almost certain there will be).

We can have this on a branch or something, so its at least easier to git diff, as I cant see the changes applied now. That branch can probably publish to another npm package like you did.

**Comment by shentao** [comment]
I agree! This was more of a PoC to see how many changes I had to make to make it work with Vue 2.x. I think what we would need to do is try to use Vue-demi again. And then we could have a single codebase for both versions.

**Pull request merged**